WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -63,9 +63,12 @@ public static bool TryParseError(JToken token, [NotNullWhen(true)] out HwiExcept
 		}
 
 		HwiErrorCode? code = null;
-		if (TryParseErrorCode(codeToken, out HwiErrorCode c))
+		if (codeToken is { })
```

Comment:
Can be but we mostly use

```suggestion
		if (codeToken is not null)
```

these days.